src/cmd/serve.rs

```diff
@@ -154,13 +154,13 @@ fn rebuild_done_handling(broadcaster: &Option<Sender>, res: Result<()>, reload_p
                         r#"
                     {{
                         "command": "reload",
-                        "path": "{}",
+                        "path": {},
                         "originalPath": "",
                         "liveCSS": true,
                         "liveImg": true,
                         "protocol": ["http://livereload.com/protocols/official-7"]
                     }}"#,
-                        reload_path
```

Comment:
I think we can write that one by hand, I don't really understand why this fixes the problem tbh

Reply:
The problem isn't the path, the path's fine, the problem is that `"D:\Some\Directory\path.html"` is not valid JSON. Because `\` like in Rust, is an escape sequence, so it fails to parse. It should become `"D:\\Some\\Directory\\path.html"`. I chose `serde_json` because it's already in the dependency hierarchy anyways (the templates use it).

Reply:
I see! Can you rebase? The tests should pass after that

Reply:
Ah yeah, didn't see the commit that snuck in there. It's done now.

Reply:
There's still something messed up in there. I'll look into it I guess. I'll just hard reset and patch.

Reply:
Alright, should be all good now.